src/app/pages/PerpetualPage/utils/pairsUtils.ts

```diff
@@ -0,0 +1,9 @@
+import { PerpetualPairDictionary } from 'utils/dictionaries/perpetual-pair-dictionary';
+import { PerpetualPair } from 'utils/models/perpetual-pair';
+
+export const perpIds = PerpetualPairDictionary.list().map(pair => pair.id);
+
+export const getPairByPerpId = (perpId): PerpetualPair | undefined =>
```

Comment:
We already have the same function in `PerpetualPairDictionary` (`getById`), delete it.